src/components/InstanceRichTooltip.tsx

```diff
@@ -0,0 +1,36 @@
+import type { FC } from "react";
+import { Tooltip } from "@canonical/react-components";
+import { InstanceRichTooltipMessage } from "./InstanceRichTooltipMessage";
+import { useIsScreenBelow } from "context/useIsScreenBelow";
+
+interface Props {
+  children: React.ReactNode;
```

Comment:
I think the interface can be simplified. It seems we always want a `ResourceLink` with the instance as a child. So we should not expect that from the caller, as we can build it here internally.

```suggestion
```